client/packages/common/src/ui/components/buttons/standard/ShrinkableBaseButton.tsx

```diff
@@ -24,28 +25,57 @@ export const StyledShrinkableBaseButton = styled(StyledBaseButton, {
         marginLeft: 8,
       }
     : {},
+
+  '&.MuiButton-root:not(:hover) .MuiSvgIcon-root': {
```

Comment:
this fixes the shrunk icon colour. Had to add the :not(:hover) so that the button didn't show as primary (i.e. the same as the background) when hovering.. and the 'primary' check so that I didn't overwrite what the dialog buttons were doing with hover states.